components/Tabs/src/private/MDCItemBar.m

```diff
@@ -71,6 +71,9 @@ @implementation MDCItemBar {
   /// Size of the view at last layout, for deduplicating changes.
   CGSize _lastSize;
 
+  /// Width of the collection view accounting for Safe Area insets at last layout.
```

Comment:
Super-nit: "Safe Area insets" should probably just be "safeAreaInsets" since it's a property name.